connect/src/spirc.rs

```diff
@@ -213,23 +213,29 @@ fn initial_device_state(config: ConnectConfig) -> DeviceState {
 fn calc_logarithmic_volume(volume: u16) -> u16 {
     // Volume conversion taken from https://www.dr-lex.be/info-stuff/volumecontrols.html#ideal2
     // Convert the given volume [0..0xffff] to a dB gain
-    // We assume a dB range of 60dB.
+    // We assume a dB range of 60dB --> 10^(60/20) = 1000 * amplitude
     // Use the equation: a * exp(b * x)
-    // in which a = IDEAL_FACTOR, b = 1/1000
-    const IDEAL_FACTOR: f64 = 6.908;
+    // in which a = 1/1000, b = ln(1000)
```

Comment:
This could just be `f64::ln(1000)`. It's not a function that's called very often, so precision trumps speed here, I'd say. It might even be so that it is compiled to a constant anyway, instead of a runtime calculation.

Reply:
If the performance suffers, another option would be:

```rust
use once_cell::sync::Lazy;

static IDEAL_FACTOR: Lazy<f32> = Lazy::new(|| f32::ln(1000.));
```